x/subscription/client/cli/tx_auto_renewal.go

```diff
@@ -12,34 +13,76 @@ import (
 
 var _ = strconv.Itoa(0)
 
+const (
+	EnableAutoRenewalFlag  = "enable"
```

Comment:
why dont we create a flag that can be either true or false?

Reply:
This is just the name of the flag that presented to the user.

I'm not sure I know what you mean...

Reply:
i mean, you are setting 2 flags for 1 boolean.
why not just use one flag?

Reply:
Fixed in b715be8644c4eddaf5744f238f7c690dd1120e3f